Automattic/jetpack
# E2E tests: use built artefacts in tests

#### Changes proposed in this Pull Request:

Add another e2e tests run that uses build workflow artefacts.
Currently restricted to trunk and release branches. 
After Build workflow completes successfully, the e2e workflow will download the build artefacts, extract them, and will use the tested plugin artefact in the e2e tests instead of running the build command.

#### Other information:


#### Jetpack product discussion
pd5faL-gm-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* This won't run until merged to trunk.
* To test locally:
	* Download the uploaded artefacts of a build workflow and extract them in the monorepo root
	* Run `.github/files/e2e-tests/prepare-plugins-for-e2e-env.sh` script. Check e2e workflow for the required env variables.
	* In the e2e project run `pnpm env:up` and then check the plugin
